tests/integration/test_poisson_pipeline.py

```
@@ -0,0 +1,103 @@
+            'exposure': exposure,
+            'age': age,
+            'time_at_risk': person_time
+        })
+        return df
+
+    def test_poisson_regression_flow(self, count_data):
+        """🔄 Test standard Poisson Regression workflow"""
+        df = count_data
+        
+        # Run Poisson Model
+        params, conf_int, pvalues, status_msg, stats_dict = run_poisson_regression(
```

Ruff (0.14.10)

Unpacked variable `conf_int` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)